python/nav/portadmin/handlers.py

```diff
@@ -25,6 +26,17 @@
 _logger = logging.getLogger(__name__)
 
 
+@dataclass
+class PoeState:
+    """Class for defining PoE states.
+    `state` is the value used on the device itself.
+    `name` is a human readable name for the state
+    """
+
+    state: Union[str, int]
+    name: str
+
+
```

Comment:
This is relevevant for cisco where u can use snmp for this, where you have internal state 1 with the name DISABLED etc,
but for juniper you have to use cmd commands for pretty much enabling and disabling, so in that case the internal state value has no purpose. So making the `state` value optional might make sense here.